Sources/AsyncConnection.swift

```diff
@@ -3,7 +3,7 @@ public protocol AsyncConnection: AsyncStream {
 }
 
 extension AsyncConnection {
-    public func open(result: (Void throws -> AsyncConnection) -> Void) throws {
+    public func open(_ result: (Void throws -> AsyncConnection) -> Void) throws {
```

Comment:
Oops, sorry then :laughing: 
I think we should avoid the use of `typealias` in the vast majority of cases.